modules/memegenBidAdapter.js

```diff
@@ -0,0 +1,127 @@
+var CONSTANTS = require('src/constants.json');
+var utils = require('src/utils.js');
+var bidfactory = require('src/bidfactory.js');
+var bidmanager = require('src/bidmanager.js');
+var adloader = require('src/adloader');
+var adaptermanager = require('src/adaptermanager');
+import {config} from 'src/config';
+
+var bidderName = 'memegen';
+/**
+ * Adapter for requesting bids from Meme Global Media Group
+ * OpenRTB compatible
+ */
+var MemeGlobalAdapter = function MemeGlobalAdapter() {
```

Comment:
won't this collide with the current adapter we have ? should change it to `MemmegenAdapter` no?